packages/amplication-server/src/core/resource/resource.service.ts

```diff
@@ -565,7 +565,7 @@ export class ResourceService {
       };
 
       const resource = await this.createService(args, user);
-      resource.tempId = modelGroupResource.tempId; //@todo: remove tempId from resource type
+      resource.tempId = newService.id; //@todo: remove tempId from resource type
```

Comment:
Perhaps this answers my previous comment. Why didn't you remove tempId completely?